packages/jest-each/src/validation.ts

```diff
@@ -74,3 +74,28 @@ export const validateTemplateTableHeadings = (
 
 const pluralize = (word: string, count: number) =>
   word + (count === 1 ? '' : 's');
+
+const START_OF_LINE = '^';
+const NEWLINE = '\\n';
```

Comment:
In order to support windows new lines, should this be `\\r\\n|\\r|\\n`? ([source](https://stackoverflow.com/a/52947649))

Reply:
I don't think so - we support using template literals for this, and about those the spec says (section 11.8.6.1 Static Semantics: TV and TRV):

> TV excludes the code units of LineContinuation while TRV includes them. <CR><LF> and <CR> LineTerminatorSequences are normalized to <LF> for both TV and TRV. An explicit EscapeSequence is needed to include a <CR> or <CR><LF> sequence.

If someone decides to throw in an explicit `'\r'` into the string, sure, but we'll consider it part of the line :upside_down_face: